### PR TITLE
Set up mujoco in setup_linux.sh and setup_osx.sh

### PR DESCRIPTION
**Pull request by gonzaiva**
The script setup_mujoco.sh is called now from the scripts to install
garage in Linux and OS X, since it's required by the packages installed
by conda.
Also, the three scripts were formatted with the style defined by Google.

### CODE REVIEW

File: scripts/setup_osx.sh [hunk 1]
```diff
@@ -19,8 +22,17 @@ brew install swig sdl sdl_image sdl_mixer sdl_ttf portmidi
 
 # Make sure that we're under the directory of the project
 cd "$(dirname "$0")/.."
+
+echo "Setting up MuJoCo..."
+/bin/bash ./scripts/setup_mujoco.sh
+if [[ "${?}" -ne 0 ]]; then
+  echo -e "\e[0;31mError: mujoco couldn't be set up\e[0m"
```
Comment (jonashen) [inline scripts/setup_osx.sh]: MuJoCo to keep it consistent

File: scripts/setup_linux.sh [hunk 2]
```diff
@@ -2,48 +2,60 @@
...
 sudo apt-get build-dep -y python-pygame
 sudo apt-get build-dep -y python-scipy
 
 # Make sure that we're under the directory of the project
 cd "$(dirname "$0")/.."
 
+echo "Setting up MuJoCo..."
+/bin/bash ./scripts/setup_mujoco.sh
+if [[ "${?}" -ne 0 ]]; then
+  echo -e "\e[0;31mError: mujoco couldn't be set up\e[0m"
```
Comment (jonashen) [inline scripts/setup_linux.sh]: MuJoCo to keep it consistent

File: scripts/setup_osx.sh (hunk 1 above)
Comment (gonzaiva) [reply scripts/setup_osx.sh]: Done

File: scripts/setup_linux.sh (hunk 2 above)
Comment (gonzaiva) [reply scripts/setup_linux.sh]: Done

File: scripts/setup_mujoco.sh [hunk 3]
```diff
@@ -1,57 +1,52 @@
...
-    read -e -p "Please enter the path to the mujoco license file [mjkey.txt]:" path
-    path=${path:-mjkey.txt}
-    eval path=$path
-    if [ ! -f $path ]; then
-        echo "No file found at $path"
-        exit 0
-    fi
-    cp $path vendor/mujoco/mjkey.txt
+if [[ ! -f vendor/mujoco/mjkey.txt ]]; then
+  read -e -p "Please enter the path to the mujoco license file [mjkey.txt]:" \
```
Comment (jonashen) [inline scripts/setup_mujoco.sh]: MuJoCo

**Pull request merged**